ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -614,6 +614,38 @@ FAN_SS_AL:
   - schedule_run_command
   - control_status
 
+FAN_SS_VSC_SRC:
+  description: "Basic Exhaust fan with start/stop, speed sensor, fail alarm, scheduled run"
+  uses:
+  - lost_power_alarm
+  opt_uses:
+  - filter_alarm
+  implements:
+  - FAN
+  - SS
+  - VSC
+  - SRC
+
+
+FAN_SS_VSC_SRC_DPM:
+  description: "Basic Exhaust fan with start/stop, speed sensor, fail alarm, scheduled run, dampers"
+  uses:
+  - lost_power_alarm
+  opt_uses:
+  - filter_alarm
+  implements:
+  - FAN
+  - SS
+  - VSC
+  - SRC
+  - DPM
+FAN_ST:
+  description: "Basic fan with flowrate status only"
+  uses:
+  - air_flowrate_status
```

Comment:
This field is undefined and generating validator [output errors](https://github.com/google/digitalbuildings/runs/6578470018?check_suite_focus=true). Please fix.

Reply:
fixed sir.